test/Esprima.Tests/Fixtures.cs

```diff
@@ -106,15 +122,33 @@ public void ExecuteTestCase(string fixture)
             {
                 sourceType = SourceType.Module;
                 expected = File.ReadAllText(moduleFilePath);
+                if (WriteBackExpectedTree)
+                {
+                    var actual = ParseAndFormat(sourceType, script, options);
+                    if (!CompareTreesInternal(actual, expected))
+                        File.WriteAllText(moduleFilePath, actual);

```

Comment:
Is the goal to overwrite the expected test result with the actual test result to see the diffs?